tests/features/corporate-blocks.feature

```diff
@@ -51,3 +51,74 @@ Feature: Corporate blocks feature
       | path  |
       | home  |
       | login |
+
+  Scenario Outline: On any pages we have correct urls in EU corporate footer block for many languages
```

Comment:
Urls are correctly shown in the EU corporate footer block when page and language are changed.